### ISSUE TITLE
New search page organisation selector does not work on keyboard

### ISSUE THREAD

**Issue by HRemonen** [UniversityOfHelsinkiCS/palaute#1283]
Cannot open the organisation selector element using the keyboard...

**Comment by Veikkosuhonen** [comment]
skill issue
